grafana/RELEASES.md

```diff
@@ -0,0 +1,6 @@
+# Release Notes for Grafana container.
+
+## 2.1.0 (9-28-2015)
```

Comment:
One issue is that we use the release version as docker image labels. More characters will make the label complex.